Add GQL status to the UNRECOGNIZED category (#151)
neo4j/docs-status-codes: modules/ROOT/pages/notifications/all-notifications.adoc

Suggestions for improvement::

> Make sure the home database has not been misspelled.

Reviewer: ```suggestion
Verify that the home database name is not misspelled.
```

======
[.include-with-GQLSTATUS-code]
======

Query::
+
[source,cypher]
----
CREATE USER linnea SET PASSWORD "password" SET HOME DATABASE Nej4
----

Returned GQLSTATUS code::
00N50

Returned Status Description::
note: successful completion - home database not found.
The database `Ne4j` does not exist.
Verify that the spelling is correct or create the database for the command to take effect.

Suggestions for improvement::

> Make sure the home database has not been misspelled.

Reviewer: ```suggestion
Verify that the home database name is not misspelled.
```

One of the labels in your query is not available in the database, make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing label name is: `Perso`)

Suggestions for improvement::

> Make sure you didn't misspell the label.

Reviewer: ```suggestion
Verify that the label is not misspelled.
```

> If nodes with the given label will be created in the future, then no change is needed.

Reviewer: ```suggestion
If you plan to create nodes with that label in the future, then no change is needed.
```

======
[.include-with-GQLSTATUS-code]
======

Query::
+
[source,cypher]
----
MATCH (n:Perso) RETURN n
----

Returned GQLSTATUS code::
01N50

Returned Status Description::
warn: unknown label.
The label `Perso` does not exist.
Verify that the spelling is correct.

Suggestions for improvement::
Make sure you didn't misspell the label.

> If nodes with the given label will be created in the future, then no change is needed.

Reviewer: ```suggestion
If you plan to create nodes with that label in the future, no change is needed.
```

m|UNRECOGNIZED
|===

======
[.include-with-GQLSTATUS-code]
======

[cols="<1s,<4"]
|===
|GQLSTATUS code
m|01N51
|StatusDescription

> a|warn: unknown relationship type. The relationship type `$rel` does not exist. Verify that the spelling is correct.

Author: ```suggestion
a|warn: unknown relationship type. The relationship type `$reltype` does not exist. Verify that the spelling is correct.
```

> Make sure you did not misspell the relationship type.

Reviewer: ```suggestion
Verify that the relationship type is not misspelled.
```

> If relationships will be created, with the given relationship type, in the future, then no change to the query is needed.

Reviewer: ```suggestion
If you plan to create relationships of this type in the future, no change is needed.
```

======
[.include-with-GQLSTATUS-code]
======
Query::
+
[source,cypher]
----
MATCH (n)-[:NonExistingType]->() RETURN n
----

Returned GQLSTATUS code::
01N51

Returned Status Description::
warn: unknown relationship type.
The relationship type `NonExistingType` does not exist.
Verify that the spelling is correct.

Suggestions for improvement::

> Make sure you did not misspell the relationship type.

Reviewer: ```suggestion
Verify that the relationship type is not misspelled.
```

> If relationships will be created, with the given relationship type, in the future, then no change to the query is needed.

Reviewer: ```suggestion
If you plan to create relationships of this type in the future, no change is needed.
```

m|UNRECOGNIZED
|===

======
[.include-with-GQLSTATUS-code]
======
[cols="<1s,<4"]
|===
|GQLSTATUS code
m|01N52
|StatusDescription

> a|warn: unknown property key. The property `$prop` does not exist. Verify that the spelling is correct.

Author: ```suggestion
a|warn: unknown property key. The property `$propkey` does not exist. Verify that the spelling is correct.
```

make sure you didn't misspell it or that the label is available when you run this statement in your application (the missing property name is: `nonExistingProp`)

Suggestions for improvement::

> Make sure you didn't misspell the property key.

Reviewer: ```suggestion
Verify that the property key is not misspelled.
```

> If the property key will be created in the future, then no change is needed to the query.

Reviewer: ```suggestion
If you plan to create that property key in the future, no change is needed.
```

======
[.include-with-GQLSTATUS-code]
======
Query::
+
[source,cypher]
----
MATCH (n:Person {nme:”Tom”}) 
RETURN n
----

Returned GQLSTATUS code::
01N52

Returned Status Description::
warn: unknown property key.
The property `nme` does not exist.
Verify that the spelling is correct.

Suggestions for improvement::

> Make sure you didn't misspell the property key.

Reviewer: ```suggestion
Verify that the property key is not misspelled.
```

> If the property key will be created in the future, then no change is needed to the query.

Reviewer: ```suggestion
If you plan to create that property key in the future, no change is needed.
```
